gflownet/envs/htorus.py

```diff
@@ -390,13 +394,14 @@ def sample_actions_batch(
         ]
         return actions, logprobs
 
+    # TODO: deprecated
```

Comment:
Is there a standard way to do this?

Reply:
Technically there is `DeprecationWarning`, but it seems to be ignored by default - I'd just do standard warning.

Reply:
I will not do this yet because actually the continuous torus inherits from the hybrid one, so things could become messier. We'll cleanup after deadlines.